### ISSUE TITLE
Add milestone property to issue event webhooks for milestoned and demilestoned actions

### ISSUE THREAD

**Issue by joeenzminger** [isaacs/github#880]
Issue event webhooks with the action "milestoned" and "demilestoned", should include a milestone property that represents the milestone that was added or removed.  

The behavior should mirror the behavior for the "labeled" and "unlabeled" actions.

As it is now, there really isn't any way to identify the milestone that was affected when an issue is demilestoned, which means we can't track project scope changes.

**Comment by Zopieux** [comment]
Indeed. While implementing an handler for that event, I did not even realize this was missing from the docs! I had to see your report to go back and fix that.

An ideal end-result would be the following API for [`issue` event](https://developer.github.com/v3/activity/events/types/#issuesevent):

Key|Type|Description
---|----|---------
assignee|object|The optional user who was assigned or unassigned from the issue.
label|object|The optional label that was added or removed from the issue.
**milestone**|**object**|**The optional milestone that was added or removed from the issue.**
